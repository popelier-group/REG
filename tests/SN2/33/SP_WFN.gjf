# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.54032000    0.00004600   -0.00039700
 H                 -1.20799000   -0.87433300   -0.54583400
 H                 -1.20807500    0.90955900   -0.48504200
 H                 -1.20772400   -0.03503100    1.02946700
 Br                 1.78431500    0.00000000    0.00010700
 Cl                -3.39385400   -0.00002100   -0.00006400

frame.wfn
