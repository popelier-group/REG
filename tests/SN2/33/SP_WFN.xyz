
6

C       -1.392830    0.028061    0.199787
H       -0.950943   -0.526500    1.018127
H       -1.203027   -0.460573   -0.747680
H       -1.037672    1.050888    0.187269
Br        1.897451   -0.038617   -0.272229
Cl       -3.227186    0.065849    0.463034
