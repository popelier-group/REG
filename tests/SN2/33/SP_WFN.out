 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.33/Gau-33757.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.33/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     33758.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.54032   0.00005  -0.0004 
 H                    -1.20799  -0.87433  -0.54583 
 H                    -1.20808   0.90956  -0.48504 
 H                    -1.20772  -0.03503   1.02947 
 Br                    1.78432   0.        0.00011 
 Cl                   -3.39385  -0.00002  -0.00006 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.540320    0.000046   -0.000397
      2          1           0       -1.207990   -0.874333   -0.545834
      3          1           0       -1.208075    0.909559   -0.485042
      4          1           0       -1.207724   -0.035031    1.029467
      5         35           0        1.784315    0.000000    0.000107
      6         17           0       -3.393854   -0.000021   -0.000064
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082813   0.000000
     3  H    1.082812   1.784928   0.000000
     4  H    1.082807   1.784937   1.784934   0.000000
     5  Br   3.324635   3.164870   3.164975   3.164349   0.000000
     6  Cl   1.853534   2.416669   2.416644   2.416676   5.178169
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.392830    0.028061    0.199787
      2          1           0       -0.950943   -0.526500    1.018127
      3          1           0       -1.203027   -0.460573   -0.747680
      4          1           0       -1.037672    1.050888    0.187269
      5         35           0        1.897451   -0.038617   -0.272229
      6         17           0       -3.227186    0.065849    0.463034
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.3939478           0.7340009           0.7340008
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       161.7621275975 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.84D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40329514     A.U. after   12 cycles
            NFock= 12  Conv=0.13D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55357-101.41792 -62.20651 -56.02433 -56.02425
 Alpha  occ. eigenvalues --  -56.02425 -10.09053  -9.33208  -8.41515  -7.09595
 Alpha  occ. eigenvalues --   -7.08725  -7.08725  -6.23821  -6.23787  -6.23787
 Alpha  occ. eigenvalues --   -2.35057  -2.35047  -2.35047  -2.35020  -2.35020
 Alpha  occ. eigenvalues --   -0.71740  -0.54877  -0.48655  -0.31355  -0.31355
 Alpha  occ. eigenvalues --   -0.28002  -0.18223  -0.18223  -0.05221  -0.05052
 Alpha  occ. eigenvalues --   -0.05052
 Alpha virt. eigenvalues --    0.10651   0.11819   0.14574   0.14575   0.17401
 Alpha virt. eigenvalues --    0.17950   0.19415   0.19415   0.20532   0.20533
 Alpha virt. eigenvalues --    0.21507   0.22146   0.22148   0.24670   0.27752
 Alpha virt. eigenvalues --    0.30230   0.30231   0.35958   0.40079   0.52996
 Alpha virt. eigenvalues --    0.52996   0.54851   0.58188   0.58188   0.61044
 Alpha virt. eigenvalues --    0.62055   0.62055   0.64456   0.75964   0.75964
 Alpha virt. eigenvalues --    0.79407   0.84177   0.88075   0.88076   0.89174
 Alpha virt. eigenvalues --    0.89176   0.94250   0.95907   0.95908   1.00346
 Alpha virt. eigenvalues --    1.00346   1.01376   1.04335   1.25464   1.25465
 Alpha virt. eigenvalues --    1.26152   1.56391   1.66944   1.66945   1.84094
 Alpha virt. eigenvalues --    1.84095   1.85601   2.10574   2.21795   2.21797
 Alpha virt. eigenvalues --    2.26408   2.26409   2.26705   2.45007   2.45007
 Alpha virt. eigenvalues --    2.54169   2.58342   2.63424   2.63425   2.65950
 Alpha virt. eigenvalues --    2.65950   3.06646   3.28605   3.86373   3.86375
 Alpha virt. eigenvalues --    4.39790   4.39792   4.40331   4.40331   4.47722
 Alpha virt. eigenvalues --    6.75159   7.80407   7.80409   7.84559   9.83402
 Alpha virt. eigenvalues --   24.02132  25.97954  25.97955  26.47874  48.06683
 Alpha virt. eigenvalues --  215.81954 290.01218 290.01219 290.051941020.82106
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.574776   0.399325   0.399327   0.399306  -0.040120  -0.085599
     2  H    0.399325   0.517192  -0.037517  -0.037505   0.003981  -0.038217
     3  H    0.399327  -0.037517   0.517209  -0.037507   0.003981  -0.038227
     4  H    0.399306  -0.037505  -0.037507   0.517122   0.003974  -0.038172
     5  Br  -0.040120   0.003981   0.003981   0.003974  35.922615   0.042027
     6  Cl  -0.085599  -0.038217  -0.038227  -0.038172   0.042027  17.152974
 Mulliken charges:
               1
     1  C   -0.647015
     2  H    0.192741
     3  H    0.192735
     4  H    0.192783
     5  Br  -0.936458
     6  Cl   0.005215
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.068757
     5  Br  -0.936458
     6  Cl   0.005215
 Electronic spatial extent (au):  <R**2>=           1294.7162
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -5.0347    Y=              0.1019    Z=              0.7222  Tot=              5.0873
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -76.4766   YY=            -44.5215   ZZ=            -45.1664
   XY=              0.6519   XZ=              4.5868   YZ=             -0.0935
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -21.0885   YY=             10.8667   ZZ=             10.2218
   XY=              0.6519   XZ=              4.5868   YZ=             -0.0935
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             42.8899  YYY=              0.3379  ZZZ=             -3.8009  XYY=              8.5509
  XXY=             -0.5443  XXZ=             -3.7180  XZZ=              8.8404  YZZ=             -1.0232
  YYZ=             -1.1303  XYZ=             -0.0682
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -1706.3637 YYYY=            -77.4010 ZZZZ=           -107.2317 XXXY=             18.2667
 XXXZ=            128.3655 YYYX=             14.1341 YYYZ=             -2.0222 ZZZX=            105.9883
 ZZZY=             -2.5144 XXYY=           -270.8304 XXZZ=           -278.8740 YYZZ=            -30.7724
 XXYZ=             -0.9510 YYXZ=             35.0674 ZZXY=              5.8488
 N-N= 1.617621275975D+02 E-N=-7.676734326619D+03  KE= 3.070002366217D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE529\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.54032,0.000046,-0.000397\H,0,-1.20799,-0.874333,-0.545834\H,0,-
 1.208075,0.909559,-0.485042\H,0,-1.207724,-0.035031,1.029467\Br,0,1.78
 4315,0.,0.000107\Cl,0,-3.393854,-0.000021,-0.000064\\Version=ES64L-G16
 RevC.01\State=1-A\HF=-3074.4032951\RMSD=1.295e-09\Dipole=-2.0014901,0.
 0000376,-0.0003549\Quadrupole=-16.1779281,8.0889049,8.0890232,-0.00014
 86,-0.0005982,0.0000014\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 Other things may change us, but we start and end with family.
 -- Anthony Brandt
 Job cpu time:       0 days  0 hours  0 minutes 26.4 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 13.7 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:03:29 2020.
